src/index/ivf_raft/ivf_raft.cuh

```diff
@@ -435,7 +447,7 @@ class RaftIvfIndexNode : public IndexNode {
                                               cudaMemcpyDefault, stream.value()));
 
                 auto indices = rmm::device_uvector<std::int64_t>(rows, stream);
-                thrust::sequence(thrust::device, indices.begin(), indices.end(), gpu_index_->size());
+                thrust::sequence(thrust::device.on(stream.value()), indices.begin(), indices.end(), gpu_index_->size());
```

Comment:
Nitpit: Any reason we can't use `res.get_thrust_policy()` here?

Reply:
No reason at all! Will update accordingly, thanks!